packages/front/src/routes/dataviz/@types/AmountsVsYear.types.ts

```diff
@@ -0,0 +1,18 @@
+import type { AmountsVsProgramRegionDto } from "dto";
+export type PartialAmountsVsProgramRegionDto = Partial<AmountsVsProgramRegionDto> & {
+    montant: number;
+    exerciceBudgetaire: number;
+};
+
+export type PartialAmountsVsProgramRegionFormatted = Partial<AmountsVsProgramRegionDto> & {
+    montant: string;
+    exerciceBudgetaire: number;
+};
+
+export enum VARS_AMOUNTS_VS_PROGRAM_REGION {
```

Comment:
En général on nomme plutôt les `enum` avec le suffix `ENUM`

```suggestion
export enum AMOUNTS_VS_PROGRAM_REGION_ENUM {
```